### PR TITLE
Feature/company info

### PR DESCRIPTION
**Pull request by Aurora12**
Added company information config. 

See `config/company.template.json` for details and `ServiceSettings.CompanyConfig` field in `config/config.json`.

For gitops-controlled deployments, this configuration is expected to be written by `configMap` settings of k8s.

This configuration file contains company information needed for the server to identify itself in emails and for certificate pinning.

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -2374,21 +2374,25 @@
...
+    "translation": "Welcome!"
+  },
   {
     "id": "api.templates.welcome_body.info2",
     "translation": "Your account details:"
   },
   {
     "id": "api.templates.welcome_body.info3",
-    "translation": "Worldr: Sovereign, Intelligent and Secure Communication Platform."
+    "translation": "You joined Worldr at {{ .Company }}: sovereign, intelligent and secure communication platform."
```
Comment (ygworldr) [inline i18n/en.json:2387]: Oxford comma man! ^_^

**Review by ygworldr — COMMENTED** [review]
My only worry is that there are no tests… I would be more comfortable with some tests.

**Comment by Aurora12** [comment]
> My only worry is that there are no tests… I would be more comfortable with some tests.

I was in the process of doing them

**Review by ygworldr — APPROVED** [review]
LGTM

**Pull request merged**